nekoyume/Assets/_Scripts/UI/Widget/System/HasIconAndButtonSystem.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.Collections;
+using Nekoyume.Game.Controller;
+using Nekoyume.L10n;
+using Nekoyume.UI.Module;
+using TMPro;
+using UnityEngine;
+
+namespace Nekoyume.UI
+{
+    public class HasIconAndButtonSystem : SystemWidget
```

Comment:
I think naming of this class is not appropriate because `HasIconAndButton` is like adjective form.
I highly suggest `IconAndButtonSystem`.